sshd/src/main/scala/ammonite/repl/sshd/SshServer.scala

```diff
@@ -0,0 +1,65 @@
+package ammonite.repl.sshd
+
+import java.io.{InputStream, OutputStream}
+import java.util.Collections
+
+import ammonite.ops.Path
+import ammonite.repl.sshd.util.Implicits
+import org.apache.sshd.agent.{SshAgentServer, SshAgent, SshAgentFactory}
+import org.apache.sshd.common.session.ConnectionService
+import org.apache.sshd.common.{FactoryManager, Channel, NamedFactory, Session}
+import org.apache.sshd.common.file.{FileSystemView, FileSystemFactory}
+import org.apache.sshd.server.PasswordAuthenticator
+import org.apache.sshd.server.keyprovider.SimpleGeneratorHostKeyProvider
+import org.apache.sshd.server.session.ServerSession
+import org.apache.sshd.{ SshServer ⇒ SshServerImpl }
+
+/**
+ * a factory to produce shell-only ssh server
```

Comment:
A more informative comment might be "A factory to construct `org.apache.sshd.SshServer`s with a nicer interface". As far as I can tell, this has nothing to do with shell-only or not-shell-only: we're just passing in `{Input, Output}-Stream`s.

Also, `shell-only` is kind of an odd phrase that doesn't really meaning anything to me. `Ammonite-REPL ssh server` would be more descriptive